cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs

```diff
@@ -1632,6 +1633,25 @@ runQueryGovState
     govState <- runQuery localNodeConnInfo target $ queryGovState eon
     writeOutput mOutFile govState
 
+runQueryFuturePParams
+  :: Cmd.QueryNoArgCmdArgs era
+  -> ExceptT QueryCmdError IO ()
+runQueryFuturePParams
+  Cmd.QueryNoArgCmdArgs
+    { Cmd.eon
+    , Cmd.commons =
+      Cmd.QueryCommons
+        { Cmd.nodeSocketPath
+        , Cmd.consensusModeParams
+        , Cmd.networkId
```

Comment:
PR needs a rebase. This is replaced by `LocalNodeConnectInfo` field now. See https://github.com/IntersectMBO/cardano-cli/pull/1034/files